GitHubCodespaces.ps1

```diff
@@ -898,6 +1074,241 @@ function Wait-GitHubCodespaceAction
     }
 }
 
+filter Add-GitHubCodespaceUser
+{
+    <#
+    .SYNOPSIS
+        Add user(s) to Codespaces billing for an organization.
+
+    .DESCRIPTION
+        Add user(s) to Codespaces billing for an organization.
+
+        The Git repo for this module can be found here: http://aka.ms/PowerShellForGitHub
+
+    .PARAMETER OrganizationName
+        Name of the Organization.
+
+    .PARAMETER UserName
+        The handle for the GitHub user account(s).
+
+    .PARAMETER AccessToken
+        If provided, this will be used as the AccessToken for authentication with the
+        REST Api.  Otherwise, will attempt to use the configured value or will run unauthenticated.
+
+    .EXAMPLE
+        Add-GitHubCodespaceUser -OrganizationName microsoft -UserName HowardWolosky
+
+    .OUTPUTS
```

Comment:
Can you list some relevant `.INPUTS` here as well?

Reply:
GitHub.User is the only relevant input I can imagine.  cba3aa480517e2c9f8bff6c46fd37b5011d07225